Source/WebCore/dom/Document.cpp

```diff
@@ -1527,6 +1553,44 @@ Ref<CSSStyleDeclaration> Document::createCSSStyleDeclaration()
     return propertySet->ensureCSSStyleDeclaration();
 }
 
+ExceptionOr<Ref<Node>> Document::importNode(Node& nodeToImport, std::optional<std::variant<bool, ImportNodeOptions>>&& arg)
```

Comment:
I recommend avoiding the `arg` abbreviation.

Reply:
Sure, spelled out "argument".